src/kibana/components/doc_table/components/table_row/cell.html

```diff
@@ -1,3 +1 @@
-<td <%= timefield ? 'class="discover-table-timefield"  width="1%"' : '' %>>
-  <%= formatted %>
-</td>
\ No newline at end of file
+<td <%= timefield ? 'class="discover-table-timefield"  width="1%"' : '' %>><%= formatted %></td>
```

Comment:
Pretty sure there are other templates that need to have the whitespace stripped from them (like src/kibana/components/doc_table/components/table_row/details.html). Consider using the [no_white_space](https://github.com/elastic/kibana/blob/master/src/kibana/utils/no_white_space.js) utility to accomplish this rather than turning them all into single-line templates.